js/src/dashboard/summary-section/performance-card.test.js

```diff
@@ -9,49 +9,25 @@ import { render } from '@testing-library/react';
 import PerformanceCard from '.~/dashboard/summary-section/performance-card';
```

Comment:
This one I guess We could delete it, since we are covering this in `summary-section.test.js`

Reply:
To some extent yes. But `summary-section.test.js` tests that the specific texts are rendered, regardless of the component and implementation underneath. While this tests the `PerformanceCard` and its capabilities.

Currently, we have 1:1 relation between them. But as time flies we could use `PerformanceCard` somewhere else as well. Then for example change `SummarySection` so it will render the link directly, and boom we no longer have `PerformanceCard` covered by tests.

So it's unit vs. integration testing.